_posts/2019-01-08-Golang.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: 'Golang Interface for TVM Runtime'
+author: Siva
+date: 2019-01-08
+---
+## Author
+
+Contributed by Huawei Technologies India Pvt Ltd, Deep Learning Group.
+
+## Scope
+
+This blog assumes the reader is familier with TVM compilation process which ends at generating 
```

Comment:
Let us try to make a brief introduction of deep learning compilation and golang without making assumption of  reader's background.

Reply:
Done